### PR TITLE
Fix linter issue

### PR DESCRIPTION
**Pull request by vmanilo**
## Changes
- fix linter - added package comment
- updated dependencies:
1. github.com/securego/gosec/v2                              2.12.0   => 2.13.1
2. gotest.tools/gotestsum                                            1.8.1     => 1.8.2
3. github.com/hashicorp/terraform-plugin-sdk/v2    2.20.0  => 2.21.0

Resolves:
- https://github.com/Twingate/terraform-provider-twingate/pull/182
- https://github.com/Twingate/terraform-provider-twingate/pull/179
- https://github.com/Twingate/terraform-provider-twingate/pull/173

### CODE REVIEW

File: twingate/client.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Package twingate is a Terraform provider for Twingate platform
```
Comment (vmanilo) [inline twingate/client.go:1]: @ekampf is this comment ok for the package level?

Comment (ekampf) [reply twingate/client.go:1]: @vmanilo  Why do we need it?

Comment (vmanilo) [reply twingate/client.go:1]: there was some linter warning about missing comment for the package

Comment (vmanilo) [reply twingate/client.go:1]: you can see it here https://github.com/Twingate/terraform-provider-twingate/actions/runs/3022789437/jobs/4862295618#step:9:169

```
Error: twingate/client.go:1:1: package-comments: should have a package comment (revive)
```

**Pull request merged**